.pipelines/wheels_linux.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   timeoutInMinutes: 120
   pool: {vmImage: 'ubuntu-latest', name: 'Linux-CPU'}
   variables:
-    CIBW_BUILD: "cp3{7,8,9}-*"
+    CIBW_BUILD: "cp3{7,8,9,10}-*"
```

Comment:
What's the reason this value isn't consistent with the macos and windows configs? Guessing it's the python versions supported but naively I would have expected we support the same versions on all platforms.

Reply:
And win32-wheel is test only and the official release packages are produced by Onebranch.official because of code sign.